andela/ah-jawans-frontend
# #166790049 Implements password reset feature

#### What does this PR do?
Implements password resetting and updating to a new password for AH users
#### Description of Task to be completed?
-create modal for the reset password
-Introduce actions and reducers
-Write password reset logic in the component folder
-Write password update logic in the component folder
-Introduce routes for the above components
-Introduce page styling
#### How should this be manually tested?
-Clone the repository locally with git clone https://github.com/andela/ah-jawans-frontend.git command.
-Check out to this branch by running git checkout ft-passwordReset-166790049.
-Install dependencies by running `npm install`.
-Update the .env file according to the .env.sample file
-Start the server with `npm start` and put this URL http://localhost:8080/passwordReset and http://localhost:8080/updatePassword in the address bar of your browser then follow the instruction of resetting your password.
#### What are the relevant pivotal tracker stories?
[#166790049](https://www.pivotaltracker.com/story/show/166790049)
#### Screenshot?
![password](https://user-images.githubusercontent.com/47073458/64017292-8a8b6680-cb29-11e9-8a06-b5998c9fac9f.gif)
